.github/workflows/autopr.yml

```diff
@@ -1,23 +1,32 @@
 on:
   push:
     branches:
-    - feature-1
+    - dev
 
 jobs:
   pull-request:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
     - name: pull-request
+      id: open-pr
       uses: repo-sync/pull-request@v2
       with:
         source_branch: ""                                 # If blank, default: triggered branch
-      git bran  destination_branch: "main"
```

Comment:
What difference is here?

Reply:
The branches name have changed in the repo